### PR TITLE
Some cleanup in the editor code

### PR DESCRIPTION
**Pull request by psfinaki**
No functional changes whatsoever.

This includes:
- renaming CodeFix folder to CodeFixes
- removing some redundant opens
- unifying naming for code fix providers
- more succinct indentation in some tests

### CODE REVIEW

File: vsintegration/src/FSharp.Editor/CodeFixes/AddInstanceMemberParameter.fs [hunk 1]
```diff
@@ -11,7 +11,7 @@ open Microsoft.CodeAnalysis.CodeFixes
 open CancellableTasks
 
 [<ExportCodeFixProvider(FSharpConstants.FSharpLanguageName, Name = CodeFix.AddInstanceMemberParameter); Shared>]
-type internal FSharpAddInstanceMemberParameterCodeFixProvider() =
+type internal AddInstanceMemberParameterCodeFixProvider() =
```
Comment (vzarytovskii) [inline vsintegration/src/FSharp.Editor/CodeFixes/AddInstanceMemberParameter.fs:14]: Mef might be confused if there's another type with the same name is loaded in codefixes (despite being in different namespace).
Not sure if it still applies.

Comment (psfinaki) [reply vsintegration/src/FSharp.Editor/CodeFixes/AddInstanceMemberParameter.fs:14]: Any way to check that? Running things locally still works.

Comment (T-Gro) [reply vsintegration/src/FSharp.Editor/CodeFixes/AddInstanceMemberParameter.fs:14]: way to check that:

Create two codefixes in different namespace, but sharing their name.

**Review by T-Gro — APPROVED** [review]

File: vsintegration/src/FSharp.Editor/CodeFixes/AddInstanceMemberParameter.fs (hunk 1 above)
Comment (psfinaki) [reply vsintegration/src/FSharp.Editor/CodeFixes/AddInstanceMemberParameter.fs:14]: Yeah, I tried this out, looks like this works. Like, I just copypasted the code of a code fix, put it into a different namespace, adjusted the names and both show up. 

image

So it doesn't matter how the types are called. That said, strings matter, if both fixes have the same "title", both will be triggered but only one will show up. This is unrelated to the changes in the PR, just good to know.

**Pull request merged**